forge-game/src/main/java/forge/game/player/PlayerProperty.java

```diff
@@ -82,81 +83,64 @@ public static boolean playerHasProperty(Player player, String property, Player s
             if (!player.hasBlessing()) {
                 return false;
             }
+        } else if (property.startsWith("damageDoneSingleSource")) {
+            String[] props = property.split(" ");
+            List<Integer> sourceDmg = game.getDamageDoneThisTurn(null, false, "Card.YouCtrl", null, source, sourceController, spellAbility);
+            int maxDmg = sourceDmg.isEmpty() ? 0 : Collections.max(sourceDmg);
+            if (!Expressions.compare(maxDmg, props[1], AbilityUtils.calculateAmount(source, props[2], spellAbility))) {
```

Comment:
A minor quibble, but could we do a substring of props[1] instead of splitting this again? To make the style more consistent with all the other comparator type stuff already out there.

And then make the Dragon Cultist `GE5`